tsweb/tsweb.go

```diff
@@ -114,3 +111,64 @@ func stripPort(hostport string) string {
 	}
 	return net.JoinHostPort(host, "443")
 }
+
+// varzHandler is an HTTP handler to write expvar values into the
+// prometheus export format:
+//
+//   https://github.com/prometheus/docs/blob/master/content/docs/instrumenting/exposition_formats.md
+//
+// It makes the following assumptions:
+//
+//   * *expvar.Int are counters.
+//   * an *expvar.Map is descended into, joining keys with
+//     underscores. So use underscores as your metric names.
+//   * an expvar named starting with "gauge_" is of type gauge.
```

Comment:
Document that these prefixes get stripped during export. I find it less objectionable if the type annotations don't escape Go.

Reply:
Done.